docs/product-spec.md

```diff
@@ -0,0 +1,100 @@
+# VxMail Product Spec
+
+## Background & Motivation
+
+The COVID-19 pandemic presents a significant challenge to the US
+General Election of 2020. How can we safely hold elections when it may
+be unsafe to congregate in a typical precinct? There is an emerging
+consensus that one important solution will be the expansion of
+vote-by-mail.
+
+Our product hypothesis is that, in states that currently have little
+vote-by-mail, the small-to-medium sized counties that process no more
+than 50,000 votes are likely to have the hardest time making the
+necessary adjustments. Many of those counties are lightly staffed, and
+many have a completely manual process for processing absentee ballot
+requests, stuffing and mailing ballots out, and finally receiving,
+sorting, verifying, extracting, and scanning the returned ballots. A
+manual process that works for a few hundred ballots quickly falls over
+for a few thousand ballots, and establishing a more automated process
+in the 6 months we have left is a huge challenge for each small county
+to undertake.
+
+We propose VxMail, a set of tools to help small counties quickly scale
+up their vote-by-mail capacity.
+
+## Key Hypothesis
+
+A number of small and some medium-sized counties have a sufficiently
+manual process for a relatively small number of vote-by-mail ballots
+today. COVID-19 will cause, for them, a shift to vote-by-mail, with 10
+times more ballots, which makes the manual process no longer
+practical&mdash;because of insufficient speed and accuracy. With
+simple COTS hardware and software, we can enable 3 election staffers
+to handle 50,000 vote-by-mail ballots assuming at least 4 weeks for
+sending out ballots prior to election day and 2 weeks for tabulation.
+
+## VxMail Components
+
+VxMail includes:
```

Comment:
Given that we're so early, can we make this plain language instead of acronyms:
-a way to automatically mail ballots to voters
-a way to process incoming ballots and check signatures
-a way to scan and tabulate ballots

Reply:
In my opinion that makes it harder to understand and talk about, not easier. You then have to talk about it by saying "that system that counts ballots, it has..." Putting a name on things makes discussion easier.

Reply:
yeah, my goal was explicitly to not name them with anything permanent, but to give them a clear consistent name *in the document* so it's easy to reference them.

I don't want to assign `Vx` names to the components yet, because it's way too soon and they may change.

Reply:
Providing/using an acronym signifies that a name is common enough to require a "simpler", "easier" name.